examples/service_explorer.py

```diff
@@ -65,7 +65,7 @@ async def main(args: Args):
             for char in service.characteristics:
                 if "read" in char.properties:
                     try:
-                        value = await client.read_gatt_char(char.uuid)
+                        value = await client.read_gatt_char(char.handle)
```

Comment:
```suggestion
                        value = await client.read_gatt_char(char)
```

Drat, after merging this, I realized, we could just do this instead. It makes more sense to pass the object directly since we already have it. Otherwise, the method call just has to look up the object again by the handle.

Reply:
Fixed in https://github.com/hbldh/bleak/commit/d65d1921ef478722ece2e610e4b6499885c6d5eb

Reply:
:facepalm: sorry, also missed that. So easy and I just followed the error message. :laughing: